Cartscreen hifi

kaeleytran opened the pull request:
hi this is the cart screen hifi and checkout she is very pretty but there is one functionality bug in da checkout screen but idk what it is oops

curesh commented on android/app/src/main/java/com/apifm/MainApplication.java:
```diff
@@ -7,7 +7,8 @@
 import com.oblador.vectoricons.VectorIconsPackage;
-import com.swmansion.gesturehandler.react.RNGestureHandlerPackage;
+import com.swmansion.gesturehandler.RNGestureHandlerPackage;
+// import com.swmansion.gesturehandler.react.RNGestureHandlerPackage;
```
Uncomment this and comment out line 10

curesh commented on src/Cart/CartProduct.js:
```diff
@@ -23,75 +23,116 @@ const styles = StyleSheet.create({
-    //    flex: 1,
-    //    flexDirection: 'column',
+    flex: 1,
+    //  flexDirection: 'column',
```
Delete this and all other extraneous comments in the files

curesh commented on src/Cart/CartScreen.js:
```diff
@@ -112,39 +134,51 @@ export default function CartScreen({ navigation }) {
-        </Title>
+    <View style={styles.entireScreen}>
+      <View style={styles.container}>
+        {/* <Title style={styles.titleText}>
```
Delete comment

curesh reviewed (dismissed):
Hi Kaylee, this feature looks great visually! There seems to be a small bug with the price calculations, where sometimes if I change the quantity of an item by doing the following
- Delete current quantity
- Enter new quantity
There's some lag in the total price calculation, where it shows the price of the cart with the current quantity deleted, instead of the new quantity. If you can fix that bug, that'd be great!

Commit:
quantities changed into objects, however total still buggy and not working

Commit:
quantities changed to objects, total calculation still buggy

Commit:
minor fixes to pr comments for final cart and checkout hifi

he-is-james committed:
fixed lag on calculating price

- created quantities list that has each item quantity and price
- simplified calc function to not call from airtable
- adjusted the quantity changes so that that it would allow for fluid changes

he-is-james committed:
Merge branch 'cart-bug-fixing' into cartscreen-hifi

Commit:
trying to pull from master

he-is-james committed:
fixed quantity adjustments in cart and produce details

he-is-james committed:
Merge branch 'cartscreen-hifi' of https://github.com/lablueprint/apifm-mobile-app into cartscreen-hifi

he-is-james committed:
Merge branch 'cart-bug-fixing' into cartscreen-hifi

he-is-james committed:
small fixes

he-is-james committed:
Merge branch 'master' into cartscreen-hifi

he-is-james committed:
removed border and adjusted deletion process

he-is-james committed:
added restrictions to cart + orders and fixed past orders navigation

he-is-james committed:
implemented delivery date
